activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -149,25 +149,39 @@ def #{method_name}=(value)             # def includes_values=(value)
 
     alias extensions extending_values
 
-    # Specify relationships to be included in the result set. For
-    # example:
+    # Specify associations to be included in the result set. A separate query
+    # is performed for each association, unless a join is required by
+    # conditions.
     #
-    #   users = User.includes(:address)
+    # For example:
+    #
+    #   users = User.includes(:address).limit(5)
     #   users.each do |user|
     #     user.address.city
     #   end
     #
-    # allows you to access the +address+ attribute of the +User+ model without
-    # firing an additional query. This will often result in a
-    # performance improvement over a simple join.
+    #   => SELECT "users".* FROM "users" LIMIT 5
```

Comment:
I think the return statements need a comment before them, they aren't being parsed as Ruby (yet) but that might be something to consider in the future.

```suggestion
    #   # => SELECT "users".* FROM "users" LIMIT 5
```

Reply:
Thanks! I've changed the examples from which I copied as well: https://github.com/rails/rails/pull/47276
It also seems we don't use the `=>` most of the time for SQL.
I'll make a separate PR for that as well.